opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py

```diff
@@ -171,6 +171,29 @@ def force_flush(self, timeout_millis: float = 10_000) -> bool:
         return True
 
 
+class InMemoryMetricExporter(MetricExporter):
```

Comment:
What is this for? It would be preferred to separate out new features/functionality that is not related to the original issue in a separate PR.

Reply:
I moved it from another test because I wanted to be able to use it as well. This is for testing purposes. Good point, I will create a pull request with this change first before merging this one.